<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDlg</class>
 <widget class="QDialog" name="ColorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>238</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>238</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Color Picker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #111;
}

QToolButton {
	background-color: #111;
	color: white;
	border-style: outset;	
	border: 1px solid #444;
}

QToolButton:hover{
	background-color: #222;
	color: #1086e2;
	border: 1px solid #444;
}

QSpinBox {
	background-color: #222;
	color:#888;
	border: 1px solid #444;
}

QSpinBox:enabled {
	background-color: #111;
	color:#FFF;
	border: 1px solid #444;
}
</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="lbl_color">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>206</width>
     <height>36</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
}

QLabel:enabled {
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_red">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>135</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #FFF;
background-color:#F00;
}</string>
   </property>
   <property name="text">
    <string>R</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #FFF;
background-color:#00F;
}</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_green">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>135</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #FFF;
background-color:#0F0;
}</string>
   </property>
   <property name="text">
    <string>G</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_blue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>135</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QDial" name="dial_b">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>65</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchTarget">
    <double>10.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dial_g">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>65</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchTarget">
    <double>10.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dial_r">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchTarget">
    <double>10.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>225</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>225</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>205</y>
     <width>216</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
